src/app/game/components/Deck/Deck.tsx

```
@@ -0,0 +1,93 @@
+        <BuyManure handleBuy={handleBuy} thisPlayer={thisPlayer} />
+        <BuyTractor
+          handleBuy={handleBuy}
+          thisPlayer={thisPlayer}
+          availableTractors={availableTractors}
+        />
+        <BuyCards thisPlayer={thisPlayer} handleBuy={handleBuy} />
+      </div>
+      <div className="flex justify-between">
+        <DecCard deckLength={deck.length} />
+        {currentPlayer.marketingCards.map((card, index) => (
+          <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)